src/Mvc/dotnet-getdocument/src/dotnet-getdocument.csproj

```diff
@@ -1,8 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>dotnet-getdocument</AssemblyName>
-    <Description>GetDocument Command-line Tool outside man</Description>
```

Comment:
Don't delete this setting. Though `$(Description)` isn't used without a package, this is nice to have as documentation